client/verta/verta/client.py

```diff
@@ -671,6 +671,10 @@ def get_or_create_registered_model(self, name=None, desc=None, labels=None, work
         id : str, optional
             ID of the registered_model. This parameter cannot be provided alongside `name`, and other
             parameters will be ignored.
+        action_type : :mod:`~verta.registry.action_type`, default :class:`~verta.registry.action_type.Unknown`, optional
+            Action type of the registered_model.
+        data_type : :mod:`~verta.registry.data_type`, default :class:`~verta.registry.data_type.Unknown`, optional
+            Data type of the registered_model.
```

Comment:
See my comment below for `create_registered_model()`

```suggestion
        action_type : :mod:`~verta.registry.action_type`, optional
            Action type of the registered_model.
        data_type : :mod:`~verta.registry.data_type`, optional
            Data type of the registered_model.
```